docs/_partner_api/changelog.md

```diff
@@ -7,6 +7,16 @@ redirect_from: /partner_api/
 
 # Changelog
 
+## 2021-07-28
+### Added `v1/profiles/me`
+The endpoint return information about the user whose `access_token` is authorized. 
+It also includes the list of available accounts. Check the endpoint
+[documentation](https://platform.zapier.com/partner_api/endpoints#get-v1profilesme) for the payload structure.
+
+### Added pagination to `v1/zaps`
+The endpoint `/v1/zaps` supports `limit`/`offset` query parameters to paginate the results.
+It now brings `count`/`next`/`previous` in the payload.
```

Comment:
`It now brings ... in the payload` sounds a bit off in this context, to me.  What are your thoughts on:

- `count`/`next`/`previous` have been added to the response payload

Reply:
For sure!!